jaxrs-api/src/main/java/javax/ws/rs/core/HttpHeaders.java

```diff
@@ -45,8 +45,8 @@
     public List<String> getRequestHeader(String name);
 
     /**
-     * Get a HTTP header as a single string value.
-     * <p/>
+     * <p>Get a HTTP header as a single string value.
```

Comment:
I may be too niggling here, but IMO we should either put both the start and end element on a line nor non of them.

So either:

```html
<p>
Foobar
</p>
```

Or:

```html
<p>Foobar</p>
```

Reply:
I agree that your PR definitely improves the situation. It's just something that came to my mind while reviewing this.